src/main/java/net/greatstart/config/WebConfig.java

```diff
@@ -0,0 +1,31 @@
+//package ua.petshop.config;
+//
+//import org.springframework.context.annotation.Bean;
+//import org.springframework.context.annotation.ComponentScan;
+//import org.springframework.context.annotation.Configuration;
+//import org.springframework.web.servlet.ViewResolver;
+//import org.springframework.web.servlet.config.annotation.DefaultServletHandlerConfigurer;
+//import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+//import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+//import org.springframework.web.servlet.view.InternalResourceViewResolver;
+//
+//@Configuration
+//@EnableWebMvc
+//@ComponentScan("ua.petshop.web")
+//public class WebConfig extends WebMvcConfigurerAdapter {
+//
+//    @Bean
+//    public ViewResolver viewResolver() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)